src/ai/backend/agent/server.py

```diff
@@ -800,6 +801,14 @@ async def server_main(
             monitor.close()
 
 
+class LogLevel(str, Enum):
+    debug = "debug"
+    info = "info"
+    warning = "warning"
+    error = "error"
+    critical = "critical"
```

Comment:
we name members of `Enum` as capital letters, like below.
```
class LogLevel(str, Enum):
  DEBUG = "debug"
  INFO = "info"
  ...
```

Reply:
Thank you for your comment! 
I wonder if there is any way to reduce duplicated code which means I want to put LogLevel class in such a file that contain class so that I can import that file to agent/server.py, agent/watcher.py and so on.